lib/handlers/sender_handler.rb

```diff
@@ -131,7 +136,7 @@ def on_container_start(container)
         sasl_allowed_mechs: @sasl_mechs,
         # Set idle timeout
         idle_timeout: @idle_timeout,
-      ).open_sender(@broker.amqp_address)
+      ).open_sender(anonymous ? nil : @broker.amqp_address)
```

Comment:
Can be for now, but for other options we will need hash `{:target => nil}` (which is now broken).

Reply:
ack